.github/workflows/test-pr.yml

```diff
@@ -10,7 +10,7 @@ concurrency:
 jobs:
 
   test-pr:
-    name: 'Bulid and test KPlutus'
+    name: 'Build and test KPlutus'
```

Comment:
You should do this change elsewhere, or I can adjust the check name name. But this affects the required status checks.

Reply:
Can leave this change for my PR: https://github.com/runtimeverification/plutus-core-semantics/pull/359